packages/0xcert-ethereum-order-gateway/src/queries/claim-personal-sign.ts

```diff
@@ -0,0 +1,17 @@
+import { Order } from '@0xcert/scaffold';
+import { OrderGateway } from '../core/gateway';
+import { createOrderHash } from '../lib/order';
+
+/**
+ * Creates order hash, signes it and returns signed order claim.
```

Comment:
Please update description here and in eth_sign so that we know what kind of signature happens.